bootstrap/js/bootstrap3/less/variables.less

```
@@ -262,7 +262,7 @@
 @dropdown-caret-color:           #000;
 
 
-//-- Z-index master list
+//-- Z-index controller list
```

I believe this is referring to the authoritative or official list, not to a list on the Jenkins controller.
```suggestion
//-- Z-index authoritative list
```